src/utils/deployFunkwhale.ts

```diff
@@ -51,7 +51,8 @@ export default async function deployFunkwhale(data: VM, profile: IProfile) {
   await deployFunkwhaleVM(grid, name, network, nodeId, domain);
 
   const info = await getFunkwhaleInfo(grid, name);
-  funkYggIp = info[0]["yggIP"];
+  funkYggIp = info[0]["planetary"];
+  console.log(funkYggIp);
```

Comment:
don't forget to clean up logs!

Reply:
Done here https://github.com/threefoldtech/grid_weblets/pull/193/commits/6dd96c7c7f71be2e8dc4674f117650503db440cb